superdwarf.lua

```diff
@@ -0,0 +1,104 @@
+-- makes units very speedy
+--@ module = true
+local help = [====[
+
+superdwarf
+==========
+Similar to `fastdwarf`, per-creature.
+Overrides units timers every game tick to 1, causing extremely fast actions
+
+Usage:
+
+``superdwarf add``: apply superdwarf to selected unit
+``superdwarf del``: remove superdwarf from selected unit
+``superdwarf clear``: clear superdwarf from all affected units
+``superdwarf list``: list all currently affected units
+``superdwarf`` or ``superdwarf help``: view this help
+]====]
+
+local repeatUtil = require('repeat-util')
+local commandArg = ({...})[1]
+
+local superId = "superdwarf"
+
+superdwarfIds = superdwarfIds or {}
+
+function AddSuperdwarf(unit)
+    if superdwarfIds[unit.id] then
+        qerror("Dwarf is already super!")
+    end
+    superdwarfIds[unit.id] = true
+    repeatUtil.scheduleEvery(superId, 1, 'ticks', function()
+        if next(superdwarfIds) == nil then
+            repeatUtil.cancel(superId)
+        else
+            for k,_ in pairs(superdwarfIds) do
+                local unit = df.unit.find(k)
+                if unit ~= nil and not unit.flags1.inactive then
+                    dfhack.units.setGroupActionTimers(unit, 1, df.unit_action_type_group.All)
+                else
+                    DeleteSuperdwarf(unit)
+                end
+            end
+        end
+
+    end)
+end
+
+function DeleteSuperdwarf(unit)
+    if superdwarfIds[unit.id] then
+        superdwarfIds[unit.id] = nil
+    else
+        qerror("Dwarf was not already super")
+    end
+    superdwarfIds[unit.id] = nil
+end
+
+function ListSuperdwarfs()
+    print("Current Superdwarfs: ")
+    for k,_ in pairs(superdwarfIds) do
+        print("[" .. k .. "] " .. dfhack.TranslateName(df.unit.find(k).name))
+    end
+end
+
+function ClearSuperdwarfs()
+    for k,_ in pairs(superdwarfIds) do superdwarfIds[k] = nil end
+end
+
+function main(argCommand)
+    if argCommand == 'help' or not argCommand then
+        print(help)
```

Comment:
To be more specific about the removal of help documentation from the scripts: You can remove your help variable entirely and replace this print command with `print(dfhack.script_help())`

Reply:
I see, fixed now